### DISCUSSION TITLE
Trying to find best config file for Intel

### DISCUSSION THREAD

**Discussion by aresstokrat** [swar/Swar-Chia-Plot-Manager#224] [Q&A]
I have 2 rigs, mostly the same: i9 9900k (16 thread), 32gb ram, 2 x 2tb nvme for temp, 1x1tb nvme for stage drive + hdds
this is my config :

` global:
  max_concurrent: 12

 - name: intel-1
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: h:\
    temporary2_directory:
    destination_directory: e:\
    size: 32
    bitfield: true
    threads: 8
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 4
    max_concurrent_with_start_early: 6
    stagger_minutes: 60
    max_for_phase_1: 4
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false`
second intel-2 is the same, but different drive letters.
Im getting this https://prnt.sc/12s6adi
when i expected 4 plotters with 2 threads,  and when after phase-1    more threads become free , i expcected 2 more to come.
maximum plots per this q with start early :6
What im doing wrong allways getting only 1 plot per 1 q name??

**Comment by Drummas** [comment]
You're assigning 8 threads to each plotter. Since your CPU only has 16 total, it only launches 2 plotters, if I'm not mistaken.
